wuzz.go

```diff
@@ -164,7 +153,7 @@ func (e *SearchEditor) Edit(v *gocui.View, key gocui.Key, ch rune, mod gocui.Mod
 func (a *App) Layout(g *gocui.Gui) error {
 	maxX, maxY := g.Size()
 	if maxX < MIN_WIDTH || maxY < MIN_HEIGHT {
-		if v, err := g.SetView("error", 0, 0, maxX-1, maxY-1); err != nil {
+		if v, err := g.SetView("error", 0, 0, maxX - 1, maxY - 1); err != nil {
```

Comment:
Why are these formatting changes in the same commit as the above functional changes? This makes for some pretty messy history, IMO.